File: backend/src/main/java/com/hilbert/features/article/model/Language.java
package com.hilbert.features.article.model;


public enum Language {
    NONE("none"),
    ENGLISH("en"),
    SPANISH("es"),
    FRENCH("fr"),
    GERMAN("de"),
    PORTUGUESE("pt"),
    ITALIAN("it"),
    JAPANESE("ja"),
    CHINESE("zh"),
    RUSSIAN("ru");

    private final String dbValue;

    Language(String dbValue) {
        this.dbValue = dbValue;
    }

    public String getDbValue() {
        return dbValue;
    }
}

File: mobile-frontend/src/app/shared/language/models/Language.ts
export enum Language {
    NONE = "NONE",
    ENGLISH = "ENGLISH",
    SPANISH = "SPANISH",
    FRENCH = "FRENCH",
    GERMAN = "GERMAN",
    PORTUGUESE = "PORTUGUESE",
    ITALIAN = "ITALIAN",
    JAPANESE = "JAPANESE",
    CHINESE = "CHINESE",
    RUSSIAN = "RUSSIAN"
}

export interface LanguageUIItem {
    value: Language,
    viewValue: string,
    flagCode: string // for flag icons
}